src/lsst/rsp/startup/services/labrunner.py

```diff
@@ -255,6 +286,9 @@ def _set_image_digest(self) -> None:
     def _expand_panda_tilde(self) -> None:
         self._logger.debug("Expanding tilde in PANDA_CONFIG_ROOT, if needed")
         if "PANDA_CONFIG_ROOT" in self._env:
+            for ev in ("USER", "PANDA_CONFIG_ROOT"):
```

Comment:
You already checked `PANDA_CONFIG_ROOT` the line before (and it's optional, so this looks weird even though it's harmless).